ngraph/core/src/op/pad.cpp

```diff
@@ -321,3 +322,27 @@ shared_ptr<Node> op::v1::Pad::clone_with_new_inputs(const OutputVector& new_args
         return make_shared<v1::Pad>(new_args.at(0), new_args.at(1), new_args.at(2), m_pad_mode);
     }
 }
+
+namespace
+{
+}
+
+bool op::v1::Pad::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
+{
+    const auto& data = inputs[0];
+    const auto& pad_value = inputs[3];
```

Comment:
This implementation is not finished yet. I have to add some checks for the number of outputs and read the pads from inputs if they can't be obtained using getters.

Reply:
@tomdol Do you have a plan to add additional checks?

Reply:
Yes, I've just added them.